tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc

```diff
@@ -1869,6 +1928,10 @@ TAO_RAL_API("ral_conv", "gpu",
             gpu::se_impl::gpu_conv_impl::ral_conv<Eigen::half, 3>);
 TAO_RAL_API("ral_qconv", "gpu",
             gpu::se_impl::gpu_conv_impl::ral_qconv<int8_t, 4>);
+TAO_RAL_API("ral_pdll_qgemm", "gpu",
+            gpu::se_impl::ral_pdll_qgemm_s8s8s8_biasadd_quant_per_tensor<2>);
+TAO_RAL_API("ral_pdll_qgemm", "gpu",
+            gpu::se_impl::ral_pdll_qgemm_s8s8s8_biasadd_quant_per_tensor<3>);
```

Comment:
`ral_gemm` only supports 2D tensors, why does qgemm need to support 3D tensors?

Reply:
If we still only want to handle the 2d case, we can insert a reshape op in the PDL file manually.

Reply:
If inserting reshape op method has no obvious advantage compared to current implementation, I prefer to maintain the status quo since these two methods actually do the same thing and it seems no significant workload reduction by inserting op in PDL file. To achieve same target by inserting op in PDL , we have to ensure the reshape op can be eliminated in later pass and it can support the 8bit tensor datatype.